ghcide/src/Development/IDE/GHC/Compat/Core.hs

```diff
@@ -703,45 +652,27 @@ instance HasSrcSpan (SrcLoc.GenLocated (SrcSpanAnn' ann) a) where
 pattern L :: HasSrcSpan a => SrcSpan -> e -> SrcLoc.GenLocated a e
 pattern L l a <- GHC.L (getLoc -> l) a
 {-# COMPLETE L #-}
-#endif
 
 -- | Add the @-boot@ suffix to all output file paths associated with the
 -- module, not including the input file itself
 addBootSuffixLocnOut :: GHC.ModLocation -> GHC.ModLocation
 addBootSuffixLocnOut = Module.addBootSuffixLocnOut
 
-#if !MIN_VERSION_ghc(9,2,0)
-splitForAllTyCoVars :: Type -> ([TyCoVar], Type)
-splitForAllTyCoVars =
-  splitForAllTys
-#endif
 
 tcSplitForAllTyVars :: Type -> ([TyVar], Type)
 tcSplitForAllTyVars =
-#if MIN_VERSION_ghc(9,2,0)
   TcType.tcSplitForAllTyVars
```

Comment:
Should we perhaps delete this simple function definition?

Its initial purpose is gone.
This raises the question again, whether we can carve out an GHC API out of our usage...

Reply:
Yes, I didn't go through doing simplification, which I should...